### ISSUE TITLE
Configuration example for BasicSeq2Seq

### ISSUE THREAD

**Issue by vikas95** [google/seq2seq#144]
Hi,

All the configuration files in example_configs folder are based on AttentionSeq2Seq model. I tried replacing AttentionSeq2Seq with BasicSeq2Seq but I am not able to run my programs. Following is my configuration file:

model: BasicSeq2Seq
model_params:
  bridge.class: seq2seq.models.bridges.ZeroBridge
  embedding.dim: 128
  encoder.class: seq2seq.encoders.BidirectionalRNNEncoder
  encoder.params:
    rnn_cell:
      cell_class: GRUCell
      cell_params:
        num_units: 128
      dropout_input_keep_prob: 1.0
      dropout_output_keep_prob: 0.95
      num_layers: 2
  decoder.class: seq2seq.decoders.BasicDecoder
  decoder.params:
    rnn_cell:
      cell_class: GRUCell
      cell_params:
        num_units: 128
      dropout_input_keep_prob: 1.0
      dropout_output_keep_prob: 1.0
      num_layers: 2
  optimizer.name: Adam
  optimizer.learning_rate: 0.0001
  source.max_seq_len: 500
  source.reverse: false
  target.max_seq_len: 128

Is this correct??

Looking forward for your response.
